content/kubeone/master/_index.en.md

```diff
@@ -178,9 +178,10 @@ a new feature or improvement, please check out our [contributing guide][2].
 
 If you want to get in touch with us and discuss about improvements and new
 features, please create a new issue on GitHub or connect with us over the
-mailing list or Slack:
+mailing list, forums, or Slack:
 
 * [`#kubeone` channel][4] on [Kubernetes Slack][10]
+* [Kubermatic forums][15]
 * [loodse-dev mailing list][9]
```

Comment:
@kron4eg Should we leave or drop this mailing list?

Reply:
>or drop this mailing list

👍

Reply:
@kron4eg Done, I'll create a PR to do the same in the KubeOne repo.